sos/report/plugins/nvidia.py

```diff
@@ -30,8 +30,17 @@ def setup(self):
             'nvlink -e'
         ]
 
+        ctk_subcmds = [
+            'cdi list',
+            '--version',
+        ]
+
+        self.add_copy_spec("/etc/cdi/nvidia.yaml")
         self.add_service_status("nvidia-persistenced")
+        self.add_service_status("nvidia-fabricmanager")
+        self.add_service_status("nvidia-toolkit-firstboot")
```

Comment:
curious, would it good to add journal for all these too. i.e. if one only wants to enable the `nvidia` plugin and nothing else?

Reply:
If these are added to the `services` tuple, it can be used for plugin enablement as well as automatically getting the journal and service status.

Reply:
Good idea. Shall I add the three services to the plugin enablement? We have its journal capture a bit different:

`self.add_journal(boot=0, identifier='nvidia-persistenced')`

But a quick test I did in a RHEL AI image showed that the nvidia-persistenced service is captured correctly. My only doubt is the 'boot=0' option.